packages/sdk-codegen/src/codeGen.ts

```diff
@@ -622,7 +623,7 @@ export interface ICodeGen {
 }
 
 export abstract class CodeGen implements ICodeGen {
-  needsRequestTypes = false
+  useNamedParameters = false
```

Comment:
whoops, this should default true in keeping with prior default
```suggestion
  useNamedParameters = true
```